From 6cec2d131e86a8a1c7453919488a4274f2582d7c Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <abderrahim.kitouni@codethink.co.uk>
Date: Tue, 6 Oct 2026 17:07:33 +0100
Subject: [PATCH 3/3] Revert "gnomeos-deps/nvidia-drivers: Update to 615.71.09"

It seems to cause issues, and I don't have access to an nvidia GPU to
debug it.

This reverts commit eb5e0ee822065e34b935b1dc6811ddb6c5ffd731.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-build-meta/-/merge_requests/5073>
---
 elements/gnomeos-deps/nvidia-drivers.bst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/elements/gnomeos-deps/nvidia-drivers.bst b/elements/gnomeos-deps/nvidia-drivers.bst
index cdefce7f7..79a99d461 100644
--- a/elements/gnomeos-deps/nvidia-drivers.bst
+++ b/elements/gnomeos-deps/nvidia-drivers.bst
@@ -4,8 +4,8 @@ sources:
 - kind: remote
   # Find the latest version from here https://www.nvidia.com/en-us/drivers/unix/ 
   # or https://download.nvidia.com/XFree86/Linux-x86_64/latest.txt for the "production" branch
-  url: nvidia:XFree86/Linux-x86_64/615.71.09/NVIDIA-Linux-x86_64-615.71.09.run
-  ref: cdceed22bbeb61248d1a6deabc2596673e3a6501698ee71ac8d2fdc28f3b70fe
+  url: nvidia:XFree86/Linux-x86_64/610.57.04/NVIDIA-Linux-x86_64-610.57.04.run
+  ref: b2e935c66b83bb00c0c857bc8e0ee0fd52de9286b40c9cc1eec29a7ce7eb116d
 - kind: local
   path: files/nvidia-config/modprobe.conf
   directory: local-config
-- 
2.56.0

